examples/mqtt/azure_iot/samplePropSharedKey.conf

```diff
@@ -0,0 +1,37 @@
+#Azure IoT Configurations
+EdgeSystemName = "TestGatewayName"
+
+# Device Keys
+device_primary_key = "<Device-Primary-Key>"
```

Comment:
add `edge_system_cert_file` and `edge_system_key_file`= `None`, so that on seeing config file user can understand the differences clearly.  But, don't refer these in example.py